headers/cub3d.h

```diff
@@ -29,14 +29,26 @@
 
 # endif
 
-// DEFINE
+// Define
 
 // Path for textures
 # define MM_GROUND	"assets/textures/minimap/ground.xpm"
 # define MM_WALL	"assets/textures/minimap/wall.xpm"
+# define MM_PLAYER	"assets/textures/minimap/player.xpm"
 
 # define MAPSDIR	"assets/maps/"
 
+// Math
+# define PI			3.14159
```

Comment:
is PI defined in math.h ?

Reply:
in `man 3 math`

```
MATHEMATICAL CONSTANTS
     In addition to the functions listed above,
     math.h defines a number of useful constants, listed below.

     CONSTANT        VALUE
     M_E             base of natural logarithm, e
     M_LOG2E         log2(e)
     M_LOG10E        log10(e)
     M_LN2           ln(2)
     M_LN10          ln(10)
     M_PI            pi
     M_PI_2          pi / 2
     M_PI_4          pi / 4
     M_1_PI          1 / pi
     M_2_PI          2 / pi
     M_2_SQRTPI      2 / sqrt(pi)
     M_SQRT2         sqrt(2)
     M_SQRT1_2       sqrt(1/2)
```
I will change PI with M_PI

Reply:
Done: 7284dabb950ec34e4b7f417b2a3539bee3da7bc2